README.md

```diff
@@ -20,18 +20,40 @@ require_once __DIR__ . '/vendor/autoload.php';
 
 ### Installation from Sources ###
 
-1. Download GridGain sources to `local_gridgain_path`
-2. Go to `local_gridgain_path/modules/platforms/php` folder
+1. Download GridGain PHP client sources to `gridgain_client_path`
+2. Go to `gridgain_client_path` folder
 3. Execute `composer install --no-dev` command
 
 ```bash
-cd local_gridgain_path/modules/platforms/php
+cd gridgain_client_path
 composer install --no-dev
 ```
 
 To use the client in your application, include `vendor/autoload.php` file, generated by Composer, to your source code, eg.
 ```
-require_once "<local_gridgain_path>/vendor/autoload.php";
+require_once "<gridgain_client_path>/vendor/autoload.php";
 ```
 
-For more information, see [GridGain PHP Thin Client documentation](https://apacheignite.readme.io/docs/php-thin-client).
\ No newline at end of file
+## Full Examples ##
+
+PHP Thin Client contains fully workable [examples](./examples) to demonstrate the main behavior of the client.
+
+### Examples Installation ###
+
+Examples are installed during [installation of the whole GridGain client from sources](#installation-from-sources).
+
+### Examples Setup and Running ###
+
+1. Run Apache GridGain server - locally or remotely - if not run yet.
```

Comment:
`Apache GridGain` - should this be `GridGain Ignite`?

Reply:
Nice one )